### PR TITLE
Test repair step listing commands

### LINKED ISSUE #35256: Write acceptance test for repair step list #34499
Related to #35207

### PR DESCRIPTION
**Pull request by skshetry**
## Description

This adds test for `occ maintenance:repair --list` command, and an error check for `occ maintenance:repair --single` (cannot be checked on happy path as it requires maintenance mode to be turned on). 

## Related Issue

Closes #35256

## How Has This Been Tested?

locally and in :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: tests/acceptance/features/bootstrap/OccContext.php [hunk 1]
```diff
@@ -600,6 +611,27 @@ public function systemConfigKeyShouldHaveValue($key, $value) {
...
 	}
 
+	/**
+	 * @Then the command output table should contain the following text:
+	 *
+	 * @param TableNode $table table of patterns to find with table title as 'table_column'
+	 *
+	 * @return void
+	 */
+	public function theCommandOutputTableShouldContainTheFollowingText(TableNode $table) {
```
Comment (skshetry) [inline tests/acceptance/features/bootstrap/OccContext.php]: Taken straight out of #35254

**Review by paurakhsharma — COMMENTED** [review]
Looks good to me :+1:

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 1 above)
Comment (individual-it) [reply tests/acceptance/features/bootstrap/OccContext.php]: - [ ] make sure we do not end up with two of the same functions before merging

**Review by individual-it — APPROVED** [review]
looks good, but wait till #35254 is merged and rebase to make sure `theCommandOutputTableShouldContainTheFollowingText` is not duplicated

File: tests/acceptance/features/bootstrap/OccContext.php (hunk 1 above)
Comment (skshetry) [reply tests/acceptance/features/bootstrap/OccContext.php]: @individual-it, that PR has codecov failure. Anyway, there will be merge conflicts, and need to be rebased. Better to merge this.

**Comment by phil-davis** [comment]
@skshetry I do not see where a backport has happened, so I added the backport-request label.
Please make the backport and link to it.

**Comment by individual-it** [comment]
backport in  #35280

**Pull request merged**